indico/modules/users/models/users.py

```diff
@@ -56,6 +56,19 @@ def from_legacy(cls, text):
         return next((x for x in cls if unicode(x.title) == text), None)
 
 
+class NameFormat(TitledIntEnum):
+    __titles__ = (_('John Doe'), _('Doe, John'), _('Doe, J.'), _('J. Doe'),
+                  _('John DOE'), _('DOE, John'), _('DOE, J.'), _('J. DOE'))
+    first_last = 0
```

Comment:
I don't think it's really possible to find great names for these options (`first_name_last_name` etc quickly becomes *very* long and verbose) - and it's not used in many places so you can always check the title/implementation to see how a format works.